Spotify.js

```diff
@@ -147,9 +147,12 @@ class Spotify {
           if (cb) cb(response.status, null, response.data)
         })
         .catch((error) => {
-          console.error(this.logMessage, "Failed to request API:", api)
+          console.error(this.logMessage, "Failed to request API:", api + "; Error Code: " + error.response.status + "; Error text: " + error.response.statusText)
```

Comment:
A quick feedback: this is somewhat redundant, because we handle extended output on line 151 during `this.handleRequestError(error)`. Also, this could end up in more problems, because `error.response` is not sure to exist.

My suggestion would be to change `this.handleRequestError(error)` and modify output for `error.response.status` and `error.response.statusText` from `_Debug` to use `console.error`.